circle.yml

```diff
@@ -1,38 +0,0 @@
-machine:
-  java:
-    version: oraclejdk8
-  environment:
-    TERM: dumb
-    GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx3g -XX:+HeapDumpOnOutOfMemoryError"'
-
-dependencies:
-  override:
-    - ./gradlew resolveConfigurations --parallel
-    - ./gradlew generateLock verifyDependencyLocksAreCurrent --stacktrace --parallel
-
-test:
-  override:
-    - ? |
-          set -eu
-          set -o pipefail
-
-          case $CIRCLE_NODE_INDEX in
-          0)
-            ./gradlew build --parallel
-            ;;
-          1)
-            ./gradlew testBouncyCastle testSun --parallel
-            ;;
-          esac
-      :
-        parallel: true
-        timeout: 1200
-  post:
-    - mkdir -p $CIRCLE_TEST_REPORTS/junit/
-    - find . -type f -regex ".*/build/test-results/.*xml" -exec cp {} $CIRCLE_TEST_REPORTS/junit/ \;
```

Comment:
I think there's a chunk of build.gradle that mentions `System.env.CIRCLE_TEST_REPORTS` which can be deleted (as this variable no longer exists in Circle2)